iphone/cli/commands/_build.js

```diff
@@ -3132,33 +3137,30 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 		},
 		legacySwift = version.lt(this.xcodeEnv.version, '8.0.0');
 
+	this.gccDefs = 'DEPLOYTYPE=' + this.deployType;
```

Comment:
Why move from an array to a string here? Feels odd to have to remember to add leading spaces to it. Also, this seems like exactly what a `Map` is for: key/value pairs. Why not hold a `Map` with the pairs of key/values and then render it as a string only when needed to write the file?